spec.bs

```diff
@@ -1505,6 +1513,46 @@ partial interface Navigator {
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Navigator>deprecatedURNtoURL(|urnOrConfig|, |send_reports|)</dfn>
+  method steps are:
+
+  1. Let |urn| be null.
+
+  1. If |urnOrConfig| is a {{USVString}}, set |urn| to |urnOrConfig|.
+
+  1. Otherwise, set |urn| to |urnOrConfig|'s [=fencedframeconfig/urn=].
+
+  1. If |urn| is <span class=XXX>TODO invalid</span>, then [=exception/throw=] a {{TypeError}}.
```

Comment:
https://github.com/WICG/fenced-frame/pull/161#discussion_r1626061123 got marked outdated, so I guess I'll re-file this comment as a stand-in for it.

Reply:
I proposed some new words here, WDYT?

Reply:
I think that sounds right? Although it would be inelegant to have to write that `i.e.,` each time we reference the concept of an invalid URN. But I suppose this is an incremental improvement so that's fine.